src/assets/main.scss

```diff
@@ -42,6 +43,10 @@ table {
   border-spacing: 0;
 }
 
+* {
+  box-sizing: border-box;
+}
+
```

Comment:
Are you sure it won't break anything?

Reply:
+-

Reply:
don't you want to rebase to the actual version of develop branch and double-check? especially on the create model page.

Reply:
No )